stable/prometheus-blackbox-exporter/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "prometheus-blackbox-exporter.fullname" . }}
+  labels:
+    app: {{ template "prometheus-blackbox-exporter.name" . }}
```

Comment:
Could you use the new labels?

Reply:
I believe this is fixed in latest commit.